src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -47,6 +48,33 @@ public enum WebAuthenticationType
     }
 
     /// <summary>
+    /// The valid values for the -SslProtocol parameter for Invoke-RestMethod and Invoke-WebRequest
+    /// </summary>
+    [Flags]
+    public enum WebSslProtocol
+    {
+        /// <summary>
+        ///  No SSL protocol will be set and the system defaults will be used.
```

Comment:
What is "system default"?  In .Net? Os?

Reply:
It is environment specific and subject to implementation changes by CoreFX, thus the comment is not specific on purpose. It is `SslProtocol.None`... which is terribly named for PowerShell users as it implies no SSL/TLS would be used (that's a C# specific terminology for the 0  field on flag enums).